pipelines/release-to-github-signed/release-to-github-signed.yaml

```diff
@@ -0,0 +1,348 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: release-to-github-signed
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton release pipeline to release binaries extracted from the image built with RHTAP, signed a checksum file, and
+    release the binaries and signed file to GitHub.
+  params:
+    - name: release
+      type: string
+      description:
+        The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution
+    - name: releaseplan
+      type: string
+      description: The namespaced name (namespace/name) of the releasePlan
+    - name: releaseplanadmission
+      type: string
+      description: The namespaced name (namespace/name) of the releasePlanAdmission
+    - name: snapshot
+      type: string
+      description: The namespaced name (namespace/name) of the snapshot
+    - name: enterpriseContractPolicy
+      type: string
+      description: JSON representation of the EnterpriseContractPolicy
+    - name: enterpriseContractPublicKey
+      type: string
+      description: Public key to use for validation by the enterprise contract
+      default: k8s://openshift-pipelines/public-key
+    - name: postCleanUp
+      type: string
+      description: Cleans up workspace after finishing executing the pipeline
+      default: "true"
+    - name: verify_ec_task_bundle
+      type: string
+      description: The location of the bundle containing the verify-enterprise-contract task
+      default: "quay.io/enterprise-contract/ec-task-bundle:8fb3bb337a971236e7f4a5a2f5b91e58f\
+        286f3bf@sha256:181f0de265fd8c425ed42da70ef86537a275a171525ec3faacf3f70ea237508a"
+  workspaces:
+    - name: release-workspace
+  # results:
```

Comment:
Should these be removed or what is the plan for them? If you are waiting to turn them on later once another task is added to the pipeline, can you add a comment saying so?

Reply:
Updated the comment